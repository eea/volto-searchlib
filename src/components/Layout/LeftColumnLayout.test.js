import React from 'react';
import renderer from 'react-test-renderer';
import LeftColumnLayout from './LeftColumnLayout';

describe('LeftColumnLayout', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LeftColumnLayout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
