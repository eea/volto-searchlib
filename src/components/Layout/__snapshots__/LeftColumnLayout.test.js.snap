// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftColumnLayout renders correctly 1`] = `
<div>
  <div
    className="ui container"
  >
    <div
      className="sui-layout-header"
    >
      <div
        className="sui-layout-header__inner"
      />
    </div>
  </div>
  <div
    className="ui container stackable two column grid body-content"
  >
    <div
      className="row"
    >
      <div
        className="three wide widescreen column"
      >
        <div />
      </div>
      <div
        className="nine wide widescreen column"
      >
        <div />
        <div />
      </div>
    </div>
  </div>
  <div
    className="ui container grid body-footer"
  >
    <div />
  </div>
</div>
`;
